### PR TITLE
add log

### PR DESCRIPTION
**Pull request by triump2020**
## What type of PR is this?

- [ ] API-change
- [ x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #16273 

## What this PR does / why we need it:
1.Add log

___

### **PR Type**
Bug fix, Enhancement

___

### **Description**
- Replaced `SetStaticTxnId` and `GetStaticTxnId` methods with `SetStaticTxnInfo` and `GetStaticTxnInfo` to store and retrieve transaction debug information.
- Added logging for successful transaction commits.
- Updated transaction commit handling to store debug information.
- Updated logging to use transaction debug information instead of transaction ID.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>mysql_cmd_executor.go</strong><dd><code>Update transaction information storage method</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
pkg/frontend/mysql_cmd_executor.go

<li>Changed <code>SetStaticTxnId</code> to <code>SetStaticTxnInfo</code> to store transaction debug <br>information.<br>

</details>
    

  </td>
  <td><a href="https://github.com/matrixorigin/matrixone/pull/16831/files#diff-af2611d5fc89704398fe09d09644efa41fec8931b395eda292f2f474f1216275">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>txn.go</strong><dd><code>Add logging for transaction commits and update transaction info </code><br><code>handling</code></dd></summary>
<hr>
      
pkg/frontend/txn.go

<li>Added logging for successful transaction commits.<br> <li> Updated transaction commit handling to store debug information.<br>

</details>
    

  </td>
  <td><a href="https://github.com/matrixorigin/matrixone/pull/16831/files#diff-3050247e22d703c6e7acd77c8ff7ff651609dc3fe102e4744700a2f7719498ba">+10/-2</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>types.go</strong><dd><code>Replace static transaction ID methods with transaction info methods</code></dd></summary>
<hr>
      
pkg/frontend/types.go

<li>Replaced <code>SetStaticTxnId</code> and <code>GetStaticTxnId</code> with <code>SetStaticTxnInfo</code> and <br><code>GetStaticTxnInfo</code>.<br> <li> Added a new field <code>txnInfo</code> to store transaction debug information.<br>

</details>
    

  </td>
  <td><a href="https://github.com/matrixorigin/matrixone/pull/16831/files#diff-5426ed8bd66760011299bcef0a9d1d552477eaae49a8451e16b8ec5bf7d7310f">+14/-8</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>util.go</strong><dd><code>Update logging to use transaction debug information</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
pkg/frontend/util.go

<li>Updated logging to use transaction debug information instead of <br>transaction ID.<br>

</details>
    

  </td>
  <td><a href="https://github.com/matrixorigin/matrixone/pull/16831/files#diff-005617216374ac58bf3cf72b81841d5cb6a17495d1d7b36ff387e294a0d6e043">+3/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: pkg/frontend/txn.go [hunk 1]
```diff
@@ -119,6 +121,8 @@ func finishTxnFunc(ses FeSession, execErr error, execCtx *ExecCtx) (err error) {
 		if execErr == nil {
 			err = commitTxnFunc(ses, execCtx)
 			if err == nil {
+				//TODO::
```
Comment (daviszhen) [inline pkg/frontend/txn.go:124]: 注释掉的都去掉吧。下同

File: pkg/frontend/util.go [hunk 2]
```diff
@@ -497,13 +497,13 @@ func logStatementStringStatus(ctx context.Context, ses FeSession, stmtStr string
 
 	}
 
+	txnInfo := ses.GetStaticTxnInfo()
 	if status == success {
-		ses.Debug(ctx, "query trace status", logutil.StatementField(str), logutil.StatusField(status.String()))
+		ses.Debug(ctx, "query trace status", logutil.StatementField(str), logutil.StatusField(status.String()), logutil.TxnIdField(txnInfo))
```
Comment (daviszhen) [inline pkg/frontend/util.go:502]: 默认是Info级别。Debug接口不回打印任何日志的。

改为info后，打印会非常多。

要判断确定sql才能改为info。

**Review by daviszhen — CHANGES_REQUESTED** [review]

File: pkg/frontend/util.go (hunk 2 above)
Comment (jensenojs) [reply pkg/frontend/util.go:502]: 加了

File: pkg/frontend/txn.go [hunk 3]
```diff
@@ -504,12 +511,16 @@ func (th *TxnHandler) commitUnsafe(execCtx *ExecCtx) error {
 	if th.txnOp != nil {
 		execCtx.ses.EnterFPrint(79)
 		defer execCtx.ses.ExitFPrint(79)
-		commitTs := th.txnOp.Txn().CommitTS
-		execCtx.ses.SetTxnId(th.txnOp.Txn().ID)
+		//commitTs := th.txnOp.Txn().CommitTS
```
Comment (daviszhen) [inline pkg/frontend/txn.go:514]: 这些不要了，都删除掉吧

File: pkg/frontend/types.go [hunk 4]
```diff
@@ -337,8 +338,10 @@ type FeSession interface {
 	ResetFPrints()
 	EnterFPrint(idx int)
 	ExitFPrint(idx int)
-	SetStaticTxnId(id []byte)
-	GetStaticTxnId() uuid.UUID
+	//SetStaticTxnId(id []byte)
```
Comment (daviszhen) [inline pkg/frontend/types.go:341]: 不要了，都删掉吧。

**Review by daviszhen — APPROVED** [review]

File: pkg/frontend/types.go (hunk 4 above)
Comment (daviszhen) [inline pkg/frontend/types.go:341]: 不要都删掉吧。

File: pkg/frontend/txn.go (hunk 3 above)
Comment (daviszhen) [inline pkg/frontend/txn.go:514]: 不要了，都删掉吧。

**Review by daviszhen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**